scilpy/tractanalysis/reproducibility_measures.py

```diff
@@ -124,6 +123,59 @@ def get_endpoints_density_map(streamlines, dimensions, point_to_select=1):
     return endpoints_map
 
 
+def get_head_tail_density_maps(streamlines, dimensions):
```

Comment:
I think we had a very similar function doing that (both extremities at the same time).
Can you try to find it (get endpoints map I think) and maybe you could call this function inside of the other to save duplicated code (then you probably would have to merge head and tail into one)